h/browser/chrome/karma.config.js

```diff
@@ -50,14 +49,24 @@ module.exports = function(config) {
       configure: function(bundle) {
         bundle
           .plugin('proxyquire-universal')
-          .require('phantom-ownpropertynames/implement', {entry: true});
+          // fix for Proxyquire in PhantomJS 1.x.
+          // See https://github.com/bitwit/proxyquireify-phantom-menace
+          .require('../../../node_modules/phantom-ownpropertynames/implement',
```

Comment:
Something changed in how the `require()` is resolved here with the dependency upgrade, with the result that the path is always resolved relative to the location of the `karma.config.js` file rather than using the normal Node require resolution algorithm. I haven't dug into the upgraded modules to find out exactly what caused the issue.

However, reading this again it occurred to me that using `require.resolve('phantom-ownpropertynames/implement')` would be a neater that hard-coding the relative location of the `node_modules` folder.